runtime/doc/options.txt

```diff
@@ -4761,14 +4761,13 @@ A jump table for the options with a short description can be found at |Q_op|.
 	height with ":set scroll=0".
 
 						*'scrollback'* *'scbk'*
-'scrollback' 'scbk'	number	(default: 10000
-				 in normal buffers: -1)
+'scrollback' 'scbk'	number	(default: -1)
 			local to buffer
 	Maximum number of lines kept beyond the visible screen. Lines at the
 	top are deleted if new lines exceed this limit.
-	Only in |terminal| buffers.
-	-1 means "unlimited" for normal buffers, 100000 otherwise.
-	Minimum is 1.
+	Minimum is -1, maximum is 100000.
+	For terminal buffers: -1 is equivalent to 10000.
+	For all other buffers: Currently not implemented and ignored.
```

Comment:
It said `Only in |terminal| buffers` already, which is more concise and less time-sensitve. We should avoid speaking about "current status" in documentation, it does not age well and makes stuff sound incomplete.